pennylane/transforms/batch_transform.py

```diff
@@ -177,7 +229,7 @@ def _wrapper(*args, **kwargs):
             interface = qnode.interface
 
             # TODO: extract gradient_fn from QNode

```

Comment:
it looks like this TODO has been done? If so, can remove

Reply:
This hasn't been done yet!